package.json

```diff
@@ -149,7 +152,9 @@
     "contracts/",
     "webtools/",
     "scripts/gsn*",
-    "webtools.webpack.js"
+    "webtools.webpack.js",
+    "helpers/",
+    "dist/"
   ],
-  "main": "src/relayclient/index.js"
+  "main": "dist/src/relayclient/index.js"
```

Comment:
is this the right "main"? we're exposing more than just the client

Reply:
I don't know, just fixed it to point to dist since it is compiled by tsc to that location. We probably need to "clean" the publish files but  I didn't filter them yet.